src/__tests__/navigator.vibrate.ts

```diff
@@ -1,3 +1,3 @@
 export const NavigatorVibrateSetup = () => {
-  navigator.vibrate = (_: VibratePattern): boolean => true;
+  navigator.vibrate = (_: number[]): boolean => true;
```

Comment:
The type of `vibrate`'s parameter should be `VibratePattern` (which is an alias for `number | number[]`), as it originally was, but now for some weird reason TypeScript only accepts `number[]`. I have no idea what's going on in here.